package.json

```diff
@@ -27,7 +27,7 @@
     "express-form-data": "^2.0.19",
     "express-session": "^1.17.3",
     "googleapis": "^110.0.0",
-    "govuk-frontend": "^4.4.1",
+    "govuk-frontend": "^4.2.0",
```

Comment:
Updated `govuk-frontend` to major version

Update in v 4.0.0 was causing the issue with the radio buttons. Removed the class `govuk-radios--inline` which resolved the issue and aligns to the GDS Radio button design component

Reply:
The template-typescript project uses v 4.5.0 see https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/package.json
Will our app behave correctly is we use that version?

Reply:
Previously running an even older version 3.7.1, We will soon look at updating to 4.5.0 once some breaking changes have been reviewed by the design team

Reply:
cool. Lets go for 4.2.0 in meantime